mxcubecore/HardwareObjects/ANSTO/prefect_flows/schemas/grid_scan.py

```diff
@@ -0,0 +1,29 @@
+from pydantic import BaseModel
+from typing import Optional, Union
+
+
+class GridScanDialogBox(BaseModel):
+    exposure_time: float
+    omega_range: float
+    detector_distance: float
+    photon_energy: float
+    hardware_trigger: bool = True
+    sample_id: Optional[str] = None
+
+
+# For more information about the tables below see
+# the SCOMPMX-569-add-strategy-table branch of the mx-data-layer-api-2
```

Comment:
I wonder if we should remove or add context for these kind of internal comments, given that we'll eventually be merging this back upstream.